metadata/metadata.ts

```diff
@@ -5840,4 +5840,68 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  Walrus: {
+    description:
+      "With his trusty tusks and love for the deep, he'll ensure you reel in an extra fish every time",
```

Comment:
This is a nit but can we be consistent with the punctuation? Just need a `.` at the end of this description. Maybe just check the other to be sure they all have punctuation.